### PR TITLE
Added resque info API and UI

### PR DESCRIPTION
**Pull request by Prevole**
Resque has been updated. It seems there is a way to do the same job did in the API. The resque update introduce shortcuts to retrieve some metrics.

### CODE REVIEW

**Comment by AlphaHydrae** [comment]
Discussed changes:
* new workers style
* remove 4th processed jobs statistic
* replace third header "Queues" with "Jobs by queue" and remove table headers

File: client/widgets/resque-stats/stats.directive.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  fetchStats();
+
+  function fetchStats() {
+    $scope.loading = true;
+
+    return api({
+      url: '/platformManagement/resqueStats'
+    }).then(function(response) {
+      if (response.data) {
```
Comment (AlphaHydrae) [inline client/widgets/resque-stats/stats.directive.js]: Superfluous `if (response.data)`.

File: features/platform_management.feature [hunk 2]
```diff
@@ -183,6 +183,30 @@ Feature: Platform management
...
+          "failed": 0
+        },
+        "workers": {
+          "total": 0,
+          "working": 0
+        },
+        "queues": []
+      }
+      """
+    And nothing should have been added or deleted
```
Comment (AlphaHydrae) [inline features/platform_management.feature]: Add at least one 403 test for resque stats.

File: client/widgets/resque-stats/stats.directive.js (hunk 1 above)
Comment (Prevole) [inline client/widgets/resque-stats/stats.directive.js]: Done

File: features/platform_management.feature (hunk 2 above)
Comment (Prevole) [inline features/platform_management.feature]: Added two tests.

**Pull request closed without merging**